db/model/Gdoc/htmlToEnriched.ts

```diff
@@ -772,7 +772,12 @@ function cheerioToArchieML(
             .with(
                 { tagName: P.union("h1", "h2", "h3", "h4", "h5", "h6") },
                 (): BlockParseResult<EnrichedBlockHeading> => {
-                    const level = parseInt(element.tagName.slice(1))
+                    let level = parseInt(element.tagName.slice(1))
+                    // We don't have any new design for h6's.
+                    // If they exist in WordPress, we switch them to h5's
```

Comment:
@ikesau there shouldn't be any h6 from the new migration code in https://github.com/owid/owid-grapher/pull/1953/files which lifts all headings one level up. What use case do you have in mind?

Reply:
Aah whoops, I hadn't seen that PR. I'll revert this change then 😛